src/Client/Web/Bit.Client.Web.BlazorUI/README.md

```diff
@@ -0,0 +1,38 @@
+﻿### BitComponent is a native high performance components for Blazor
+[![NuGet Version](https://img.shields.io/nuget/v/Bit.Client.Web.BlazorUI.svg?style=flat)](https://www.nuget.org/packages/Bit.Client.Web.BlazorUI/) ![Nuget](https://img.shields.io/nuget/dt/Bit.Client.Web.BlazorUI.svg)
+
+BitComponent is a set of native and high performance Blazor UI controls. This is implemented in C# and this does not depend on or wrap existing JavaScript frameworks or libraries.
+
+**Why choose Bit Blazor Components?**
+- BitComponent is free
+- BitComponent is open source
+- BitComponent is high performance
+- BitComponent is native blazor components
+- BitComponent is supported in **server-side** and **client-side** (WASM) Blazor
+
+to use BitComponent, please follow these steps:
```

Comment:
we need to change this line just like this:
To use the Bit components, please follow these steps: